README.md

```diff
@@ -1,2 +1,2 @@
 # feast-operators
-Charmed Operators for Feast feature store
+This bundle contains Charmed Operators for [Feast](https://feast.dev/) feature store.
```

Comment:
```suggestion
This repository contains Charmed Operators for [Feast](https://feast.dev/) feature store.
```
?
just a suggestion but I 'm not really sure. Is this a supposed to be viewed as a bundle?

Reply:
yes it would be a bundle that's pluggable with kubeflow, similar to mlflow bundle

Reply:
ACK makes sense